packages/create-block/package.json

```diff
@@ -20,7 +20,7 @@
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
 	"engines": {
-		"node": ">=12",
+		"node": ">=14",
 		"npm": ">=6.9"
```

Comment:
Do we need to bump the minimum `npm` version as well? I think the one that goes with Node `14.0.0` is `npm@6.14.4`.

Reply:
I'm fine with that, too 👍🏻